packages/chains/src/bsc/redemptionStrategies.ts

```diff
@@ -100,6 +100,14 @@ const redemptionStrategies: { [token: string]: [RedemptionStrategyContract, stri
     RedemptionStrategyContract.UniswapLpTokenLiquidator,
     underlying(assets, assetSymbols.HAY),
   ],
+  [underlying(assets, assetSymbols["sAMM-HAY/BUSD"])]: [
+    RedemptionStrategyContract.UniswapLpTokenLiquidator,
+    underlying(assets, assetSymbols.HAY),
```

Comment:
why is the output token HAY here? for flash loan liquidations we will likely need to repay with a stable asset, like BUSD or WBNB